Searching Algorithms/Binary Search Algorithm/C++/binary-search.cpp

```diff
@@ -4,19 +4,17 @@ Theory of Programming
 
 Binary Search Algorithm
 http://theoryofprogramming.com/2016/11/04/binary-search-algorithm/
-GitHub - https://github.com/salman-bhai/theoryofprogramming
-Code Contributor - <salman-bhai>
+GitHub - https://github.com/VamsiSangam/theoryofprogramming
+Code Contributor - Praveen Mylavarapu (github.com/praveenmylavarapu, saipraveen.ml)
```

Comment:
All contributors are important :stuck_out_tongue_closed_eyes: . You could change this to -

```
Code Contributors - <salman-bhai>
                  - Praveen Mylavarapu (github.com/praveenmylavarapu, saipraveen.ml)
```